# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/openrc/hotfix-destdir.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- openrc-0.63.3/tools/meson_final.sh.vanilla	2026-09-02 01:15:27.146761221 +0300
+++ openrc-0.63.3/tools/meson_final.sh	2026-09-02 01:16:01.997065291 +0300
@@ -9,4 +9,4 @@
 if [ "${os}" != linux ]; then
 	install -d "${DESTDIR}/${rc_libexecdir}"/init.d
 fi
-install -m 644 "${MESON_BUILD_ROOT}/src/shared/version" "${DESTDIR}/${rc_libexecdir}"
+install -m 644 "${MESON_BUILD_ROOT}/src/shared/version" "${rc_libexecdir}"

